<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FindAndReplaceDlg</class>
 <widget class="QDialog" name="FindAndReplaceDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find and Replace</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="replaceLineEdit" />
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="findLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Replace w&amp;ith:</string>
         </property>
         <property name="buddy" >
          <cstring>replaceLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Find &amp;what:</string>
         </property>
         <property name="buddy" >
          <cstring>findLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="caseCheckBox" >
         <property name="text" >
          <string>&amp;Case sensitive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wholeCheckBox" >
         <property name="text" >
          <string>Wh&amp;ole words</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>231</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="moreFrame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="backwardsCheckBox" >
          <property name="text" >
           <string>Search &amp;Backwards</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="regexCheckBox" >
          <property name="text" >
           <string>Regular E&amp;xpression</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreNotesCheckBox" >
          <property name="text" >
           <string>Ignore foot&amp;notes and endnotes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="findButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>&amp;Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moreButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>&amp;More</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>findLineEdit</tabstop>
  <tabstop>replaceLineEdit</tabstop>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>wholeCheckBox</tabstop>
  <tabstop>backwardsCheckBox</tabstop>
  <tabstop>regexCheckBox</tabstop>
  <tabstop>ignoreNotesCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FindAndReplaceDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>344</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>362</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>moreFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>291</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
